build/tools/SharedCodeGenerator/Program.cs

```diff
@@ -107,7 +107,7 @@ private static void GenerateSupportedPlatformsReadmeFile(string repoDir)
             {
                 sw.WriteLine("# Supported platforms and versions");
                 sw.WriteLine();
-                foreach (var subDirPath in Directory.GetDirectories(platformsDir))
+                foreach (var subDirPath in Directory.GetDirectories(platformsDir).OrderBy(n => n))
                 {
                     var subDirInfo = new DirectoryInfo(subDirPath);
                     var platformSubDirPaths = Directory.GetDirectories(subDirPath);
```

Comment:
Should we also enforce order for this call to `GetDirectories`?

Reply:
i think it would be ok since the `subDirPath` is sorted, but just to be safe i'll add sorting here as well. thanks!